scripts/circleci/setup-angular-snapshots.py

```diff
@@ -0,0 +1,70 @@
+"""
+    Script that sets up the Angular snapshot github builds. We set up the snapshot builds by
```

Comment:
Can you expand this comment to explain why this script is in python compared to everything else in the repo?

Reply:
Done!